src/core/server/http/__tests__/http_config.test.ts

```diff
@@ -25,6 +25,14 @@ test('has defaults for config', () => {
   expect(httpSchema.validate(obj)).toMatchSnapshot();
 });
 
+test('accepts valid hostnames', () => {
```

Comment:
Can you add a case for `localhost` so we have a test case for the default value?

Reply:
:heavy_check_mark: